app/views/pwa/service-worker.js

```diff
@@ -0,0 +1,26 @@
+// Add a service worker for processing Web Push notifications:
```

Comment:
Has this been generated somehow? You prefer to have such "default" files added to the code base, don't you?

Reply:
Yes, this has been generated. At first, I was tempted to remove it, so that we don't keep (and deliver) this boiler-plate code to clients.

However, it is also a valuable documentation on getting started with web workers and also the default in new Rails application. The route to deliver this file exists (and is working; `pwa_service_worker`), but it is not used anywhere, so that the impact of this file is rather minimal. Still, with the aim to be up-to-date and following the Rails best practices, I then thought it might be acceptable to leave the file in the repo and use it on demand (e.g., as part of the `manifest.webmanifest`.